package.json

```diff
@@ -47,7 +48,7 @@
     "@types/classnames": "^2.2.10",
     "@types/node": "^14.0.27",
     "@types/react": "^16.9.44",
-    "@types/theme-ui": "^0.3.6",
+    "@types/theme-ui": "^0.3.7",
```

Comment:
Maybe this is why we're having problems with theme-ui typings. Try downgrading to `0.3.1` (our current version of `theme-ui`, and see if it works well by removing our `index.d.ts`

Reply:
It didn't work

Reply:
oka